Havtorn/Source/Editor/Windows/InspectorWindow.cpp

```diff
@@ -110,6 +110,8 @@ namespace ImGui
 	{
 		if (selectedEntity->HasComponent(componentType))
 		{
+			// TODO.NR/AG: Extract Collapsing Header and Remove Component button logic and move it here
```

Comment:
Just leaving a note from Discord discussion,
We can use GetComponentTypeString here to get the 'const char* label' parameter for ImGui::CollapsingHeader. Although we'll have to decide if we want the "component" part of the string or not :)